Sources/View/ReusableView.swift

```diff
@@ -25,11 +25,21 @@ public extension UICollectionView {
         register(cellType, forCellWithReuseIdentifier: cellType.reuseIdentifier)
     }
 
+    func register<T: UICollectionViewCell>(_ cellType: T.Type)
+    where T: ReusableView, T: NibView {
```

Comment:
We can probably use the new `&` operator on conformances: `T: ReusableView & NibView` 😉